lib/aggregators/blocks.ts

```diff
@@ -24,7 +24,7 @@ class BlocksAggregator extends Aggregator {
         { total: blocks.length }
       );
     }
-    return Promise.map(blocks, async (_block) => {
+    return BluebirdPromise.map(blocks, async (_block: any) => {
```

Comment:
What's the advantage of using BlueBirdPromise over native promises here?

Reply:
Native Promise does not have a `map` function -- the `Promise` global variable was previously overridden / decorated.